models/intel/text-recognition-0015/accuracy-check.yml

```diff
@@ -0,0 +1,133 @@
+evaluations:
+  - name: text-recognition-0015
+    module: custom_evaluators.custom_text_recognition_evaluator.TextRecognitionWithAttentionEvaluator
+    module_config:
+      model_type: SequentialTextRecognitionModel
+      network_info:
+        max_seq_len: "24"
+        custom_label_map:
+          0: "<s>"
+          1: ""
+          2: "</s>"
+          3: "?"
+          4: "0"
+          5: "1"
+          6: "2"
+          7: "3"
+          8: "4"
+          9: "5"
+          10: "6"
+          11: "7"
+          12: "8"
+          13: "9"
+          14: "a"
+          15: "b"
+          16: "c"
+          17: "d"
+          18: "e"
+          19: "f"
+          20: "g"
+          21: "h"
+          22: "i"
+          23: "j"
+          24: "k"
+          25: "l"
+          26: "m"
+          27: "n"
+          28: "o"
+          29: "p"
+          30: "q"
+          31: "r"
+          32: "s"
+          33: "t"
+          34: "u"
+          35: "v"
+          36: "w"
+          37: "x"
+          38: "y"
+          39: "z"
+          40: "A"
+          41: "B"
+          42: "C"
+          43: "D"
+          44: "E"
+          45: "F"
+          46: "G"
+          47: "H"
+          48: "I"
+          49: "J"
+          50: "K"
+          51: "L"
+          52: "M"
+          53: "N"
+          54: "O"
+          55: "P"
+          56: "Q"
+          57: "R"
+          58: "S"
+          59: "T"
+          60: "U"
+          61: "V"
+          62: "W"
+          63: "X"
+          64: "Y"
+          65: "Z"
+
+      launchers:
+        - framework: dlsdk
+
+      datasets:
+        - name: SVT_recognition
+          preprocessing:
+            - type: bgr_to_gray
+              cast_to_float: false
+            - type: resize
+              dst_width: 256
+              dst_height: 64
+          lowercase: true
```

Comment:
lowercase is part of what?

Reply:
lowercase is part of dataset and goes to evaluator

Reply:
Maybe it would be better to pass to as evaluator parameter (on the same level as `model_type`)

Reply:
why it is placed between preprocessing and metric?
I think better to mode it before run cofiguration or add as parameter in place, where it is really used (e.g. converter if annotation should be provided in lowercase or adapter if it should be applied to decoded sequence

Reply:
Moved to evaluator parameters